inbm-vision/Changelog.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ## NEXT - ?
 
 ### Added
+ - HSD 15012036803 - Added fix for few telemetry data of the OTA update is not published to Thingsboard cloud
 ### Fixed
```

Comment:
This should be under fix instead of add.  If it's something that is marked as a bug, it should be a fix and not an add.